larch/retrievers/sinequa.py

```diff
@@ -214,6 +214,7 @@ def __init__(
         search_parameters: Optional[Dict] = None,
         sql: Optional[str] = None,
         neural_search: bool = True,
+        text_column_name="text",
```

Comment:
Can you add type-hints also. Also follow proper format  `text_column_name: str = "text"`

Reply:
ok

Reply:
Just push the update